kubernetes/src/main/java/io/kubernetes/client/ApiClient.java

```diff
@@ -768,10 +768,10 @@ public String selectHeaderAccept(String[] accepts) {
      *
      * @param contentTypes The Content-Type array to select from
      * @return The Content-Type header to use. If the given array is empty,
-     *   JSON will be used.
+     *    or matches any, JSON will be used.
      */
     public String selectHeaderContentType(String[] contentTypes) {
-        if (contentTypes.length == 0) {
```

Comment:
I don't think we can do this, we'll need to make this change in the upstream swagger-codegen and I don't think we can land it there.

The right thing to do is to fix the Kubernetes swagger to not have this.